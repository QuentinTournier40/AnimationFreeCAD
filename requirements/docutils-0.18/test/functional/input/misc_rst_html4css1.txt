===============================
Additional tests with html4css1
===============================

.. include:: data/section_titles.txt
