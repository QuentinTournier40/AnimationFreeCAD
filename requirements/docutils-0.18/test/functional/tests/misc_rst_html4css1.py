# Source and destination file names.
test_source = "misc_rst_html4css1.txt"
test_destination = "misc_rst_html4css1.html"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "html4css1"

# Settings
# test for encoded attribute value in optional stylesheet name:
settings_overrides['stylesheet'] = 'foo&bar.css, html4css1.css'
settings_overrides['stylesheet_path'] = ''
# local copy of stylesheets:
# (Test runs in ``docutils/test/``, we need relative path from there.)
settings_overrides['stylesheet_dirs'] = ('.', 'functional/input/data')
