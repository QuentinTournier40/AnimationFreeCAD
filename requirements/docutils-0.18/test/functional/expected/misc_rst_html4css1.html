<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.18: http://docutils.sourceforge.net/" />
<title>Additional tests with html4css1</title>
<link rel="stylesheet" href="foo&amp;bar.css" type="text/css" />
<link rel="stylesheet" href="functional/input/data/html4css1.css" type="text/css" />
<link rel="stylesheet" href="functional/input/data/math.css" type="text/css" />
</head>
<body>
<div class="document" id="additional-tests-with-html4css1">
<h1 class="title">Additional tests with html4css1</h1>

<div class="section" id="section-heading-levels">
<h1>Section heading levels</h1>
</div>
<div class="section" id="level-1">
<h1>Level 1</h1>
<p>Nested sections</p>
<div class="section" id="level-2">
<h2>Level 2</h2>
<p>reach at some level</p>
<div class="section" id="level-3">
<h3>Level 3</h3>
<p>(depending on the document class and output format)</p>
<div class="section" id="level-4">
<h4>level 4</h4>
<p>a level</p>
<div class="section" id="level-5">
<h5>level 5</h5>
<p>that is not supported by the output format.</p>
<div class="section" id="level-6">
<h6>level 6</h6>
<p>Unsupported in LaTeX and HTML5
(HTML5 reserves the 1st level for the document title).</p>
<div class="section" id="level-7">
<h6 class="h7">level 7</h6>
<p>Unsupported in HTML4.</p>
<div class="section" id="level-8">
<h6 class="h8">level 8</h6>
<p>Unsupported in ODT.</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="section" id="section-titles-with-inline-markup">
<span id="references"></span><h1>Section titles with inline markup</h1>
<div class="section" id="emphasized-h2o-x-2-and-references">
<h2><em>emphasized</em>, H<sub>2</sub>O, <span class="formula"><i>x</i><sup>2</sup></span>, and <a class="reference internal" href="#references">references</a></h2>
</div>
<div class="section" id="substitutions-fail">
<h2>Substitutions work</h2>
<p>Note, that the &quot;reference name&quot; for this section is derived from the
content <em>before</em> substitution. This may be exploited to get intelligible
IDs after <a class="reference external" href="https://docutils.sourceforge.io/docs/ref/rst/directives.html#identifier-normalization">identifier normalization</a> of the section's reference name.</p>
</div>
</div>
</div>
</body>
</html>
